packaging/build_deps.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
```

Comment:
u and pipefail flags are also good to have

Reply:
u flag cannot be used in this script because the /opt/ros/foxy/setup.bash script has some variables which are not set and we get unbound error. This script is not under our control.